lib/openshift/machine_set.rb

```diff
@@ -18,7 +18,7 @@ def available_replicas(user: nil, cached: true, quiet: false)
     def ready?(user: nil, cached: false, quiet: false)
       result = {}
       result[:success] = (desired_replicas(user: user, cached: cached, quiet: quiet) ==
-        available_replicas(user: user, cached: cached, quiet: quiet))
+        available_replicas(user: user, cached: true, quiet: quiet))
```

Comment:
Update in step implemention ? I though it should be on step definition, that's line 18

Reply:
This is intended. Both methods call `raw_resource` method, the second call could use the cached resource to save time. This is @akostadinov's suggestion when I first implemented it.

Reply:
Ack